File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int, input().split())

if a == b == c:
  print(f"{10000 + (a * 1000)}")
elif a == b or a == c:
  print(f"{1000 + (a * 100)}")
elif b == c:
  print(f"{1000 + (b * 100)}")
else:
  m = max(a, b, c)
  print(m * 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int A = Integer.parseInt(st.nextToken());
        int B = Integer.parseInt(st.nextToken());
        int C = Integer.parseInt(st.nextToken());

        int m = 0;

        if (A == B && B == C) {
            m = 10000 + (A * 1000);
        } else if (A != B && A != C && B != C) {
            m = Math.max(A, Math.max(B, C)) * 100;
        } else if (A == B || B == C) {
            m = 1000 + (B * 100);
        } else {
            m = 1000 + (A * 100);
        }
        System.out.println(m);
    }
}
